package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "systemjs",
   "version": "6.2.6",
+  "main": "dist/system-node.cjs",
```

Comment:
I think it would be nice to include the `exports` field as well as the main, with this value:

```js
  "exports": {
    "node": "./dist/system-node.cjs",
    "default": "./dist/system.min.js"
  }
```